crate/crash/repl.py

```diff
@@ -111,83 +111,17 @@ def append(self, string):
-        "ntile", "null", "nullif", "nulls", "numeric", "object", "octet_length",
-        "of", "off", "offset", "old", "on", "only", "open", "optimize", "option",
-        "or", "order", "ordinality", "out", "outer", "output", "over",
-        "overlaps", "overlay", "pad", "parameter", "partial", "partition",
-        "partitioned", "partitions", "path", "percent", "percent_rank",
-        "percentile_cont", "percentile_disc", "period", "persistent", "plain",
-        "portion", "position", "position_regex", "power", "precedes",
-        "preceding", "precision", "prepare", "preserve", "primary",
-        "primary key", "prior", "privileges", "procedure", "public", "range",
-        "rank", "read", "reads", "real", "recursive", "ref", "references",
-        "referencing", "refresh", "regr_avgx", "regr_avgy", "regr_count",
-        "regr_intercept", "regr_r2", "regr_slope", "regr_sxx",
-        "regr_sxyregr_syy", "relative", "release", "rename", "repeat", "repository",
-        "reset", "resignal", "restore", "restrict", "result", "return",
-        "returns", "revoke", "right", "role", "rollback", "rollup", "routine",
-        "row", "row_number", "rows", "savepoint", "schema", "schemas", "scope",
-        "scroll", "search", "second", "section", "select", "sensitive",
-        "session", "session_user", "set", "sets", "shards", "short", "show",
-        "signal", "similar", "size", "smallint", "snapshot", "some", "space",
-        "specific", "specifictype", "sql", "sqlcode", "sqlerror", "sqlexception",
-        "sqlstate", "sqlwarning", "sqrt", "start", "state", "static",
-        "stddev_pop", "stddev_samp", "stratify", "stratify", "strict", "string",
-        "submultiset", "substring", "substring_regex", "succeedsblob", "sum",
-        "symmetric", "system", "system_time", "system_user", "table", "tables",
-        "tablesample", "temporary", "text", "then", "time", "timestamp",
-        "timezone_hour", "timezone_minute", "to", "token_filters", "tokenizer",
-        "trailing", "transaction", "transient", "translate", "translate_regex",
-        "translation", "treat", "trigger", "trim", "trim_array", "true",
-        "truncate", "try_cast", "type", "uescape", "unbounded", "under", "undo",
-        "union", "unique", "unknown", "unnest", "until", "update", "upper",
-        "usage", "user", "using", "value", "value_of", "values", "var_pop",
-        "var_samp", "varbinary", "varchar", "varying", "versioning", "view",
-        "when", "whenever", "where", "while", "width_bucket", "window", "with",
-        "within", "without", "work", "write", "year", "zone"]
 
     def __init__(self, cmd):
         self.cmd = cmd
+        self.keywords = self._populate_keywords()
+
+    def _populate_keywords(self):
+        success = self.cmd._exec("SELECT word FROM pg_catalog.pg_get_keywords()")
+        if success:
+            return [i[0] for i in self.cmd.cursor.fetchall()]
+        else:
+            return []
```

Comment:
@mfussenegger I'm not so sure about returning an empty list of the `pg_get_keywords()` call fails. Do you think a fallback list of a few most common SQL keywords would be better?

Reply:
> Do you think a fallback list of a few most common SQL keywords would be better?

Yes I think that would be good to have, for people who use a new crash to connect to old CrateDB clusters.